tests/runTests.sqf

```diff
@@ -95,17 +96,21 @@ diag_log format ["    %1", _currentDirectory];
                     {
                         private _mode = _x select 0;
                         private _test = _x select 1;
+                        private _desc = if(_test isEqualType []) then { _test select 0 } else { str(_test) };
+                        private _code = if(_test isEqualType []) then { _test select 1 } else { _test };
```

Comment:
More nested arrays?
How about just check if count _x > 2. And have description be last element in array?